lib/plugins/features/movies/controller.js

```diff
@@ -36,14 +38,23 @@ exports.list = async (query) => {
   }).fetchAll({ withRelated: ['locations'] });
 };
 
-exports.addLocationToMovie = async (movie_id, location_id) => {
+exports.addLocationToMovie = async (movie_id, location_name) => {
```

Comment:
you can pass in the payload as the second parameter here then access that location name from the payload. Also, remember to camelCase the JS variables here!